### PR TITLE
build: smoke test

### PR DESCRIPTION
**Pull request by mistermalm**
## **Describe pull-request**  
Scheduled install and build for all packages

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3483-smoke-test](https://tegel.atlassian.net/browse/CDEP-3483)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to github actions. Check that it runs and completes when scheduled

## **Checklist before submission**
- [ ] All existing tests pass
- [ ] Not breaking production behavior
- [ ] `npm run build-all` without errors

## **Additional context**  
Might need to change schedule to run more often, for test purposes

### CODE REVIEW

File: .github/workflows/smoketest-all.yml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: npm install
+
+      - name: Core - Run build
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: npm run build
+
+      - name: Core - Publish (dry-run)
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: npm publish --tag latest --dry-run
```
Comment (timrombergjakobsson) [inline .github/workflows/smoketest-all.yml:46]: 💯

File: .github/workflows/smoketest-all.yml [hunk 2]
```diff
@@ -0,0 +1,200 @@
+name: Scheduled Release Smoke Test
+
+on:
+  schedule:
+    # Runs every other Monday at 9:00 AM UTC
+    - cron: '0 9 */14 * *'
```
Comment (timrombergjakobsson) [inline .github/workflows/smoketest-all.yml:6]: Have we decided on a day to run it, or is it more of a suggestion? I think Monday or Tuesday is good. Tuesday the latest to have time to act on potential problems!

**Comment by timrombergjakobsson** [comment]
Nice @mistermalm @ckrook, do we think is good as a start, or should we add a post hook to Teams? I think that can eventually be done in a later PR, what do you think?

File: .github/workflows/smoketest-all.yml (hunk 2 above)
Comment (mistermalm) [reply .github/workflows/smoketest-all.yml:6]: I think mabye we can run it every day?

Comment (timrombergjakobsson) [reply .github/workflows/smoketest-all.yml:6]: Sure!

**Review by nathalielindqvist — APPROVED** [review]
This looks like a great start! Like Tim mentioned a Teams-hook could be pretty nice to track how the tests turn out, but then again we know that they are running so we could keep track of them on GitHub for now if we don't want to add the hook to this PR

**Comment by mistermalm** [comment]
> Nice @mistermalm @ckrook, do we think is good as a start, or should we add a post hook to Teams? I think that can eventually be done in a later PR, what do you think?

We decided to make another task for that

**Review by ckrook — APPROVED** [review]

**Review by timrombergjakobsson — APPROVED** [review]

File: .github/workflows/smoketest-all.yml [hunk 3]
```diff
@@ -0,0 +1,200 @@
+name: Scheduled Release Smoke Test
+
+on:
+  schedule:
+    # Runs every morning at 05:00
+    - cron: '0 5 * * *'
```
Comment (theJohnnyMe) [inline .github/workflows/smoketest-all.yml:6]: Nice!

**Comment by theJohnnyMe** [comment]
@mistermalm  - Do you think we need to have "on-failure" jobs for all steps?

**Comment by mistermalm** [comment]
> @mistermalm - Do you think we need to have "on-failure" jobs for all steps?

Probably not, since this is a dry-run we would want to catch all errors we can, so that we will not miss anything just because it ends early. I think we can always run the whole run since we are not pushing or publishing anything

**Comment by theJohnnyMe** [comment]
> > @mistermalm - Do you think we need to have "on-failure" jobs for all steps?
> 
> Probably not, since this is a dry-run we would want to catch all errors we can, so that we will not miss anything just because it ends early. I think we can always run the whole flow since we are not pushing or publishing anything. @timrombergjakobsson @theJohnnyMe

Sounds good to me, then ship it ! 🚀

**Review by theJohnnyMe — APPROVED** [review]

**Pull request merged**